README.md

```diff
@@ -1,219 +1,251 @@
-update the Atom Feed module config to produce these URLs. To do this, from the
-OpenMRS Ref. App home page, choose the "Atomfeed" option (this should appear
-once the Atom Feed module is installed) and click on "Load Configuration". From
-the top menu, choose the file provided in this repository at
-[`utils/fhir2_atom_feed_config.json`](utils/fhir2_atom_feed_config.json) and
-click "Import".
-
-To check the above two steps, create a new Patient (or an Observation) and
-verify that you can see that Patient (or Observation) at the aforementioned
-FHIR URL. Then check the Atom Feed URL corresponding to your change, e.g.,
-
-`http://localhost:9016/openmrs/ws/atomfeed/patient/1`
-
-OR
-
-`http://localhost:9016/openmrs/ws/atomfeed/observation/1`
-
-## Set up the Atom Feed client side database
-The Atomfeed client requires a database to store failed event and marker information, and we'll
-use MySQL for this purpose. If you don't have an available MySQL service, you can start one up using docker:
-`docker run -e "MYSQL_ROOT_PASSWORD=root" -p 127.0.0.1:3306:3306 --name=atomfeed-db -d mysql/mysql-server:latest`
-
-Now you should have MySQL running on the default port 3306, and can run:
-
-`mysql --user=USER --password=PASSWORD < utils/dbdump/create_db.sql`
-
-This will create a database called `atomfeed_client` with required tables (the
-`USER` should have permission to create databases). If you want to change the
-default database name `atomfeed_client`, you can edit [`utils/dbdump/create_db.sql`](
-utils/dbdump/create_db.sql) but then you need to change the database name in
-[`src/main/resources/hibernate.default.properties`](
-src/main/resources/hibernate.default.properties) accordingly.
-
-## Create the sink FHIR store and BigQuery dataset
-To set up GCP project that you can use as a sink FHIR store:
+## Prerequisites
+
+-   An [OpenMRS](https://openmrs.org) instance with the latest version of the
+    [FHIR2 Module](https://addons.openmrs.org/show/org.openmrs.module.openmrs-fhir2-module)
+    installed.
+    -   There is a [OpenMRS Reference Application](#run-openmrs-using-docker)
+        image with these prerequisites and demo data you can use to try things
+        out.
+-   A target output for the data. Supported options are Apache Parquet files
+    (not supported by Atom Feed mode), FHIR API resources, and
+    [Google Cloud Platform FHIR stores](https://cloud.google.com/healthcare/docs/how-tos/fhir).
+    -   You can use our [HAPI FHIR server](#run-hapi-fhir-server-using-docker)
+        image for testing FHIR API targets.
+    -   [Learn how to create a compatible GCP FHIR store](#create-a-google-cloud-fhir-store-and-bigquery-dataset).
+
+### Installing the FHIR2 Module
+
+**To install via web**: Log in to OpenMRS as an administrator, go to **System
+Administration > Manage Modules > Search from Addons**, search for the `FHIR2`
+module, and install. Make sure to install the FHIR2 module, not FHIR.
```

Comment:
Also in the above line: `FHIR2` -> `fhir2` because that is how the module is named.
```suggestion
module, and install. Make sure to install the `fhir2` module, not `fhir`.
```